pkg/scheduler/extender/extender.go

```diff
@@ -101,6 +106,9 @@ func (e *Extender) FilterHandler(w http.ResponseWriter, req *http.Request) {
 		}
 		return
 	}
+	defer func() {
+		_ = req.Body.Close()
```

Comment:
Does it make sense to print error here?

Reply:
If you only read from it, the error of Close is irrelevant.

Reply:
Not sure what do you mean. You have TCP session established which should be closed to avoid resources leak. If close operation failed error message must be logged.

Reply:
Your note is related to `file` reading, but this is http body they're different.

The `req.Body.Close()` function returns an error in order to fulfill the `io.Closer` interface; the function doesn't necessarily return an error.

Reply:
Ok